bin/check-running-job

```diff
@@ -1,33 +1,55 @@
 #!/bin/bash
 
 function check_running_job {
+    [[ $# -eq 0 ]] && echo "Error: no app provided" && exit 1
+
+    # Get wait arugment if supplied
+    if [[ -n $2 && $2 == '--wait' ]]; then
+        wait_for_app='true'
+    fi
+
     # Input any app name
-    app_to_check=$1
+    if [[ -n "$1" ]]; then
+        if [[ $1 == '--wait' ]]; then
+            echo "Error: no app provided"
+            exit 1
+        fi
+        app_to_check=$1
+    fi
 
-    # Install jq
-    apt install -y jq
+    # Check jq
+    if ! command -v jq &> /dev/null
+    then
+        apt install -y jq
+    fi
```

Comment:
Was there some issue with `jq` failing to install because it was already installed?  Or are you trying to prevent unnecessary updates to `jq`?  Or is this test for something else?

Reply:
No, just is quicker to check if it's there if the script is looping while waiting for a deploy.